my-app/src/components/Card/Avatar/Avatar.css

```diff
@@ -5,3 +5,9 @@
   border-radius: 50%;
   margin-right: 24px;
```

Comment:
looks good. 
could you try to change css to scss? it is great tool widely used and might improve readability of the code.